spec/rails_helper.rb

```diff
@@ -1,6 +1,3 @@
-require 'coveralls'
```

Comment:
@tansaku I removed coveralls from here and I didn't added simplecov since I think that by having it on features and specs is enough, but maybe I'm wrong. What do you think?

Reply:
Here you will find the [related info](https://github.com/colszowka/simplecov#merging-results) in what I based my decision.